CMakeLists.txt

```diff
@@ -32,14 +32,16 @@ include_directories(
 find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
+find_package(pybind11 REQUIRED)
```

Comment:
Aren't we already doing this is a subdirectory?

Reply:
Since the image.app include the reader_h5.h and reader_h5.h require the pybind so it is added here. Or another way is to add an option to decide if the reader_h5.h needed to be included in source code.